src/views/noMatch/noMatch.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+
+const noMatch = () => {
+  return (
+    <>
+      <div className="noMatch__container">
```

Comment:
There doesn't seem to be a defined class with this name, so we can remove this reference to it

Reply:
I forgot to include the scss file in the last commit, I've added it to the commit i just pushed.